frontend/doc-site/04__Analyze Public Data/4_3__CellGuide Documentation/4_3_1__CellGuide API.mdx

```diff
@@ -0,0 +1,34 @@
+# CellGuide API
+
+CellGuide currently has an experimental API that allows users to programmatically fetch the computational marker genes data underlying the marker gene tables in each CellGuide page.
```

Comment:
Given conversations with @pablo-gar and @niknak33 , I understand we believe it is likely that this API will change in the future as part of a larger initiative. If that is correct, I suggest we message the probability of change explicitly in addition to the experimental label.

Reply:
>>> Please note that the CellGuide API is currently in an experimental phase and is subject to change as we continue to expand its offerings. Future versions of the API may introduce new endpoints or modify existing ones to enhance functionality and user experience. However, to ensure a seamless integration for our users, all previous versions of the API will remain operational even after new versions are released. This means that no immediate updates will be required from your side. Nonetheless, we encourage users to stay informed about new releases and improvements.